testing/PIMS.Tests.Automation/Drivers/BrowserDriver.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Microsoft.Extensions.Configuration;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace PIMS.Tests.Automation.Drivers
+{
+    public class BrowserDriver
+    {
+        private readonly Lazy<IWebDriver> currentWebDriverLazy;
+        private readonly Lazy<IConfiguration> configurationLazy;
+        private bool _isDisposed;
+        private readonly bool closeBrowserOnDispose;
+        private readonly bool runAutomationHeadless;
+
+        public BrowserDriver()
+        {
+            currentWebDriverLazy = new Lazy<IWebDriver>(CreateWebDriver);
+            configurationLazy = new Lazy<IConfiguration>(ReadConfiguration);
+            closeBrowserOnDispose = Configuration.GetValue("CloseBrowserAfterEachTest", true);
+            runAutomationHeadless = Configuration.GetValue("RunHeadless", true);
+        }
+
+        public IWebDriver Current => currentWebDriverLazy.Value;
+
+        public IConfiguration Configuration => configurationLazy.Value;
+
+        private IWebDriver CreateWebDriver()
+        {
+            var options = new ChromeOptions();
+            if (runAutomationHeadless)
+            {
+                options.AddArguments("window-size=1920,1080", "headless");
+            }
+            else
+            {
+                options.AddArguments("start-maximized");
+            }
+
+            var chromeDriver = new ChromeDriver(ChromeDriverService.CreateDefaultService(), options);
```

Comment:
Cool! could we get an edge driver as well? That would help catch browser compatibility issues.